install/kubernetes/istio-rbac-alpha.yaml

```diff
@@ -3,20 +3,23 @@
 # Also helps to enable logging on apiserver 'wrap' to see the URLs.
 # Each RBAC deny needs to be mapped into a rule for the role.
 # If using minikube, start with '--extra-config=apiserver.Authorization.Mode=RBAC'
+#
+# NOTE: If deploying istio to a namespace other than 'default' then change the
+# ClusterRoleBinding namspace target appropriately.
```

Comment:
please delete this file (rbac alpha) entirely, it will rot and is not supported

Reply:
Lets not delete this until we know for sure that RBAC alpha is not needed. I see rbac/v1alpha1 in all kube 1.7* clusters when I do `kubectl api-versions`.

Reply:
where do you see rbac alpha ?
$ kubectl api-versions|grep rbac
rbac.authorization.k8s.io/v1beta1

didn't we already decided to use 1.7+ and rbac (beta) long ago

Reply:
```
shriram@shrirams-mbp:~/istio.io/istio $ kubectl api-versions
apiextensions.k8s.io/v1beta1
apiregistration.k8s.io/v1beta1
apps/v1beta1
authentication.k8s.io/v1
authentication.k8s.io/v1beta1
authorization.k8s.io/v1
authorization.k8s.io/v1beta1
autoscaling/v1
batch/v1
batch/v2alpha1
certificates.k8s.io/v1beta1
config.istio.io/v1alpha2
extensions/v1beta1
networking.k8s.io/v1
policy/v1beta1
rbac.authorization.k8s.io/v1alpha1
rbac.authorization.k8s.io/v1beta1
settings.k8s.io/v1alpha1
storage.k8s.io/v1
storage.k8s.io/v1beta1
v1
```

Reply:
I am merely proposing to postpone the culling, until we know for sure that RBAC alpha is not needed by anyone in our user base. If you look at the mailing lists, you would see that @frankbu and @andraxylia had been helping people troubleshoot really strange RBAC errors on clusters where both alpha and beta was enabled. Sometimes rbac-beta worked and sometimes, rbac-alpha. So, I am not yet buying the fact that Kube 1.7 clusters have only RBAC beta enabled.

Reply:
I would rather remove it in the next commit.
I also need to remove the extra rbac-beta , which has now been incorporated in istio.yaml. I leave it there for the e2e tests, too avoid too many changes in one PR.

PTAL and approve.

Reply:
there is no test nor use nor support of/for -alpha files  (compare https://github.com/istio/istio/search?utf8=✓&q=istio-rbac-alpha&type= vs beta) - I'd rather have the cleanup contained in any PR that touches those files than delayed but if you insist not to and do link that followup PR it's ok maybe

is what you add to istio-auth.yaml exactly the same as rbac-beta ? (it's a bit hard to see on the diff)